src/main/java/com/enterprise/fundraisermanager/dto/Fundraiser.java

```diff
@@ -1,23 +1,49 @@
 package com.enterprise.fundraisermanager.dto;
 
 import lombok.Data;
-import java.awt.*;
-import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.HashMap;
-import java.util.HashSet;
+import org.springframework.format.annotation.DateTimeFormat;
 
+import javax.persistence.*;
+import java.util.Calendar;
+import java.util.List;
+/**
+ * This is the Model Layer for fundraiser. Data that can be stored to a database is represented by this class. Each instance is associated with an entry in a database table.
+ * @author
+ */
+@Entity
+@Table(name="fundraiserdb")
 public @Data
 class Fundraiser {
-    private Integer id;
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
+    @Column(name="activeStatus")
     private Boolean activeStatus;
+
+    @Column(name="name")
     private String name;
+
+    @Column(name="description")
     private String description;
-    private Image homeImage;
-    private Goal goal;
-    private ArrayList<Tier> tierList;
+
+    @Column(name="homeImage")
+    private String homeImage;
+
+    @OneToMany(mappedBy = "goals")
+    private List<Goal> goals;
+
+    @OneToMany(mappedBy = "tierList")
+    private List<Tier> tierList;
+
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME, pattern = "yyyy-MM-dd HH:mm:ss")
+    @Column(name="startDate")
     private Calendar startDate;
+
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME, pattern = "yyyy-MM-dd HH:mm:ss")
+    @Column(name="endDate")
     private Calendar endDate;
```

Comment:
I changed your application's model layers (DTO) so that data that may be stored in a database is represented in this class. Each instance corresponds to a row in a database table. The @Entity annotation indicates that the class is an entity that is mapped to a database table. A lightweight persistence domain object is called an entity. A table in a relational database is typically represented by an entity, and each entity instance corresponds to a row in that table. Although entities can employ helper classes, the entity class is the principal programming artifact of an entity. The name of the database table to be utilized for mapping is specified by the @Table annotation.
 
I added @Id .This annotation provides the entity's primary key. The @GeneratedValue annotation specifies the primary key generation strategy to be utilized. The @Column annotation is used to add the name of a column to a table in a MySQL database. @DateTimeFormat declares that a field or method parameter should be formatted as a date or time, which is needed by your project description.

This is a helpful link: https://spring.io/guides/gs/accessing-data-jpa/

Reply:
Wait until a database is present before adding annotations.  
If you add annotations that are not used, the next programmer will make an incorrect assumption: that this is stored.